lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -383,6 +398,32 @@ IPV_IDENTITY_REUSE_PAGE_TEST:
       targetState:
         PYI_NEW_DETAILS_PAGE
 
+PYI_F2F_DELETE_PAGE:
+  response:
+    type: page
+    pageId: pyi-f2f-delete-details
+  parent: END_JOURNEY
+  events:
+    next:
+      targetState:
+        PYI_F2F_CONFIRM_DELETE_DETAILS_PAGE
+    end:
+      targetState:
+        IPV_IDENTITY_PENDING_PAGE_TEST
+
+PYI_F2F_CONFIRM_DELETE_DETAILS_PAGE:
+  response:
+    type: page
+    context: f2f
+    pageId: pyi-confirm-delete-details
+  events:
+    next:
+      targetState:
+        RESET_IDENTITY_USER_INITIATED
```

Comment:
I think we need this to route to a new state for F2F that routes to the pyi-details-deleted page with context `f2f` (as the content is slightly different) ?

Reply:
Yeah we do have the page pyi-details-deleted.njk made dynamic already but I think we'll need a new state for it to pass the context in? as atm this will go to the non-F2F pyi-details-deleted page